src/Chirp.Core/Services/IChirpService.cs

```diff
@@ -15,5 +15,7 @@ public interface IChirpService
                 Task FollowAuthor(Guid authorId, Guid authorToFollowId);
 
                 Task UnfollowAuthor(Guid authorId, Guid authorToUnfollowId);
+
+                Task DeleteAuthor(Guid authorId);
```

Comment:
Maybe rename DeleteAuthor on ChirpService layer, to ForgetAuthor? The method does more than delete.